tjreports/plugins/system/plg_system_sendemail/bulksendemail.js

```diff
@@ -0,0 +1,282 @@
+			});
+
+			// Create editor field
+			 tinyMCE.execCommand( 'mceAddEditor', false, "email-message");
+
+			// alert(values.join (", "));
+		}
+		catch (err) {
+			console.log(err.message);
+		}
+	},
+	validate: function () {
+
+		jQuery('textarea').each(function (index, ta) {
+			var $ta = jQuery(ta);
+			var emailMessageValue = $ta.val();
+		});
+
+		var emailSubjectValue = jQuery("#email-subject").val();
+		var emailMessageValue = jQuery("#email-message").val();
+		var invalidCount = 0;
+
+		jQuery("#errorMessage").empty();
+
+		if (!emailSubjectValue)
+		{
+			invalidCount = 1;
+
+			jQuery('#email-subject').addClass("invalid");
+			jQuery('#email-subject-label').addClass("invalid");
+
+			var errormsg = '<div class="alert alert-error alert-danger"><button type="button" data-dismiss="alert" class="close">×</button><h4 class="alert-heading"></h4><div>Field required: Subject</div></div>';
+			jQuery("#bulkEmailModal").find("#errorMessage").append(errormsg);
+		}
+		else
+		{
+			jQuery('#email-subject').removeClass("invalid");
+			jQuery('#email-subject-label').removeClass("invalid");
+		}
+
+		if (!emailMessageValue)
+		{
+			invalidCount = 1;
+
+			jQuery('#email-message').addClass("invalid");
+			jQuery('#email-message-label').addClass("invalid");
+
+			var errormsg = '<div class="alert alert-error alert-danger"><button type="button" data-dismiss="alert" class="close">×</button><h4 class="alert-heading"></h4><div>Field required: Message</div></div>';
+			jQuery("#bulkEmailModal").find("#errorMessage").append(errormsg);
+		}
+		else
+		{
+			jQuery('#email-message').removeClass("invalid");
+			jQuery('#email-message-label').removeClass("invalid");
+		}
+
+		if (invalidCount)
+		{
+			return false;
+		}
+
+		jQuery('#preload').show();
+		jQuery('div').find("#emailPopup").addClass('is-progress');
+		jQuery('#send-email').attr("disabled", true);
+		var chunk_size = 5;
+
+		var postData = jQuery("#emailTemplateForm").serializeArray();
+
+		var emailContent = postData.splice(0,2);
+
+		// Remove duplicate values from email array
+		postData = postData.map(JSON.stringify).reverse() // convert to JSON string the array content, then reverse it (to check from end to begining)
+		.filter(function(item, index, postData){ return postData.indexOf(item, index + 1) === -1; }) // check if there is any occurence of the item in whole array
+		.reverse().map(JSON.parse) // revert it to original state
+
+		// Chuck of 10 emails
+		var emails = postData.map( function(e,i){
+			return i%chunk_size===0 ? postData.slice(i,i+chunk_size) : null;
+		}).filter(function(e){ return e; });
+
+		var batchCount = emails.length;
+
+		jQuery.each(emails, function (i, batch) {
+			var emailData = batch.concat(emailContent);
+
+			// Call the send email function
+			tjutilitysendemail.send(emailData, i, batchCount);
+		});
+	},
+	send: function (emailData, index, batchCount) {
+		index ++;
+
+		var url = "index.php?option=com_ajax&plugin=plg_System_Sendemail&format=json";
```

Comment:
@MeghaBiranje prepend the site domain to URL. You can use Joomla.getOptions('system.paths').base